ceaser.py

```diff
@@ -0,0 +1,30 @@
+"""This script prompts a user to enter a message to encode or decode
+using a classic Caeser shift substitution"""
+
+import string
+
+def encode(message, offset):
+    """ Encode function """
+    encoded = ''
+    letters = string.ascii_letters + string.punctuation + string.digits
+    for letter in message:
+        if letter == ' ':
```

Comment:
Right now, your tests never allow for this condition to be true, so line 12 is never tested. Add another test to your ceaser_test.py script that covers this part of the code.